src/clj_puppetdb/http.clj

```diff
@@ -29,9 +30,9 @@
 
 (defn- make-client-common
   [^String host opts]
-  (let [opts (assoc opts :as :stream)
```

Comment:
I think the idea is that we now pass this down explicitly from the middleware like so: https://github.com/scottyw/pe-console-middleware/blob/8415e399f48fd7d4064c941dca05ac2035d07cf7/src/puppetlabs/pe_console_middleware/cm/pdb.clj#L22